app/retail/templates/emails/email_campaign_day_2.txt

```diff
@@ -0,0 +1,32 @@
+Day 2: Using Gitcoin’s Issue Explorer
+
+Hey {{firstname}},
+
+By this point, you might have poked around the Gitcoin Issue Explorer (https://gitcoin.co/explorer?q=) to see the open source projects you could make money by contributing! If interested in doing work, here’s how:
+
+Start Work
+Click ‘Start Work’ to kick off a message to the repo-maintainer that you’re interested in the issue
+Respond on the Github thread with your intended approach for the bounty
+
+[GIF of Start Work]
```

Comment:
i think you can remove these tags from the txt version

Reply:
right! fixed in e4c7ca3